.github/workflows/nightly-test-parent.yml

```diff
@@ -0,0 +1,116 @@
+name: Nightly Test Parent
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  push:
+    branches:
+      - main
+    paths:
+      - "python/sglang/version.py"
+  workflow_dispatch:
+
+concurrency:
+  group: nightly-test-parent-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  # Text model accuracy tests
+  nightly-test-text-accuracy-2-gpu-h100:
+    if: github.repository == 'sgl-project/sglang'
+    uses: ./.github/workflows/nightly-test-text-accuracy-2-gpu-h100.yml
```

Comment:
Let's also create a folder called nightly and call all these workflows in the subfolder for better organizing them